Netkan/Transformers/CurseTransformer.cs

```diff
@@ -39,6 +39,10 @@ public IEnumerable<Metadata> Transform(Metadata metadata, TransformOptions opts)
                 // Look up our mod on Curse by its Id.
                 var curseMod = _api.GetMod(metadata.Kref.Id);
                 var versions = curseMod.All();
+                if (opts.SkipReleases.HasValue)
+                {
+                    versions = versions.Take(opts.SkipReleases.Value);
```

Comment:
Shouldn't this be `Skip` instead of `Take`? If I use something like `--skip-releases 3 --releases 5`, this would reduce the `versions` list to the first three elements before we try to grab 5 of them.

Reply:
Ugh, that's what you get for only runnning it with SpaceDock and writing tests for GitHub...
Yeah, you're right, this has to be `Skip` of course.

Reply:
Changed in both cases :+1: